src/__tests__/u-chartStorage.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * @jest-environment jsdom
+ */
+
+const chartStorage = require('../lib/chartStorage')
+require('@testing-library/jest-dom')
+
+test('using saveChart to save data to localStorage', () => {
+    // Arrange- Creating test data to be saved to localStorage
+    const testChart = ('test data')
+    
+    // Act- Saving test data using saveChart and fetching chart data
+    chartStorage.saveChart(testChart)
+    const loadAllSavedCharts = window.localStorage.getItem("savedCharts")
+
+    // Assert- expecting the localStorage chart data to contain test data
+    expect(loadAllSavedCharts).toContain('test data')

```

Comment:
Yeah, it seems like localStorage uses some weird syntax where the data is stored as /"test data/". I could add the slash to the assertion, but I felt like that made it seem like I was leading the test to pass.

I forgot to ask abt the test data... I just asked now

Reply:
Thanks for asking about the test data!

That caught me by surprise, so I checked out this branch to see what you were saying about data being stored as `\"test data\"`. It looks like this is actually Jest escaping the second set of quotation marks, because changing it to the following worked:
```js
expect(loadAllSavedCharts).toBe('["test data"]')
```

So several of these tests could probably be changed to a stricter matcher, but I'll let you decide if that's really important.